trade-finance-manager-api/api-tests/v1/acbs/acbs.api-test.js

```diff
@@ -63,20 +58,25 @@ describe('acbs controller', () => {
   });
 
   describe('createACBS', () => {
-    it('should return false if deal if no banks exists', async () => {
-      const result = await acbsController.createACBS({
-        dealSnapshot: {
-          ...MOCK_DEAL,
-          bank: null,
-        },
-      });
+    afterEach(() => {
+      jest.clearAllMocks();
+    });
+
+    it.each(invalidIds)('should return false if an invalid Mongo Object deal id has been supplied', async (dealId) => {
```

Comment:
This will appear in the test suite as the same test description for each case which would be hard to debug. Could we change this to be `should return false if the deal id is '%s' (not a valid mongo object id)`, async (dealId)`

Reply:
Good spot, have added string substitution.